sqe-dtos/TextFragment.cs

```diff
@@ -26,6 +26,10 @@ public TextFragmentDataListDTO(List<TextFragmentDataDTO> textFragments)
 			this.textFragments = textFragments;
 		}
 
+		public TextFragmentDataListDTO() : this(null)
```

Comment:
Why not add give the parameter of the first constructor a default null value instead of a whole new constructor?

Reply:
This is funny.  I think I see the work of some autotooling here, nice catch.  In fact I think you can just pass null to the TextFragmentDataListDTO(List<TextFragmentDataDTO> textFragments) constructor, since `List<T> myVar = null` is valid.